### PR TITLE
Warn about all missing keys in plugin.cfg

### PR DESCRIPTION
**Pull request by merumelu**
Previously only the one encountered first was reported.

### CODE REVIEW

File: editor/editor_plugin_settings.cpp [hunk 1]
```diff
@@ -96,45 +96,62 @@ void EditorPluginSettings::update_plugins() {
...
+				String name = cf->get_value("plugin", "name");
+				String author = cf->get_value("plugin", "author");
+				String version = cf->get_value("plugin", "version");
+				String description = cf->get_value("plugin", "description");
+				String script = cf->get_value("plugin", "script");
+
+				TreeItem *item = plugin_list->create_item(root);
+				item->set_text(0, name);
+				item->set_tooltip(0, TTR("Name:") + " " + name + "\n"
+									+ TTR("Path:") + " " + path + "\n"
```
Comment (akien-mga) [inline editor/editor_plugin_settings.cpp]: `clang-format` doesn't like this custom alignment: https://travis-ci.org/godotengine/godot/jobs/578954211

Comment (merumelu) [reply editor/editor_plugin_settings.cpp]: Hm, I wonder why the pre commit hook did not complain about it. :/ What version of clang-format should I have?

Comment (akien-mga) [reply editor/editor_plugin_settings.cpp]: 8.0, but earlier versions should have caught it too. Check if the pre-commit hook actually (it uses `which clang-format` to locate it, maybe that's not working for you?).

**Review by akien-mga — APPROVED** [review]

File: editor/editor_plugin_settings.cpp (hunk 1 above)
Comment (merumelu) [reply editor/editor_plugin_settings.cpp]: That wasn't it but I actually only copied the `pre-commit-clang-format` to .git/hooks and not `pre-commit` as well like the readme said. :sweat_smile:

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**